package.json

```diff
@@ -75,24 +74,32 @@
     "uglify-es": "3.3.9"
   },
   "dependencies": {
-    "@babel/parser": "7.8.4",
-    "@babel/types": "7.8.3",
+    "@ethersproject/abi": "5.5.0",
+    "@ethersproject/base64": "5.5.0",
+    "@ethersproject/basex": "5.5.0",
+    "@ethersproject/bignumber": "5.5.0",
+    "@ethersproject/bytes": "5.5.0",
+    "@ethersproject/cli": "5.5.0",
+    "@ethersproject/hash": "5.5.0",
+    "@ethersproject/keccak256": "5.5.0",
+    "@ethersproject/pbkdf2": "5.5.0",
+    "@ethersproject/properties": "5.5.0",
+    "@ethersproject/random": "5.5.0",
+    "@ethersproject/rlp": "5.5.0",
```

Comment:
We should not need to include the `rlp` package

Reply:
`rlp` package is removed